invenio_vocabularies/contrib/names/datastreams.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-Vocabularies is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+from ...datastreams.datastreams import BaseDataStream
+from ...datastreams.errors import NoNameError
+from ...datastreams.readers import TarReader
+from ...datastreams.transformers import XmlTransformer
+from ...datastreams.writers import PrinterWriter
+
+
+class OrcidXmlDataStream(BaseDataStream):
```

Comment:
Do we want to make this explicit? I think this can be obtained using the configuration in the yaml and creating a `BaseDataStream`. We just need to add maybe the `OrcidXmlTransformer` to the config. (see `conftest.py`).